### ISSUE TITLE
Voice Over - Header not announced if there is a link in the cell

### ISSUE THREAD

**Issue by kata-lin** [Legal-and-General/canopy#121]
When I navigate through the table (VO+ right arrow key) using VO, the 'Fund' column header is not read out before the content of the cell.

When we removed the  `lg-table-cell` (a Canopy directive) from the cell, it works normally and also if we remove the link and just have text, it reads it as expected.

Screen recording in the following ticket: https://udcjira.lgnet.co.uk/browse/DWP-7469

**Comment by thatguynamedandy** [comment]
Hi @kata-lin thanks for raising, I was just wondering is the bug present in either of the storybook examples?

http://localhost:58976/?path=/story/components-table--standard
http://localhost:58976/?path=/story/components-table--detail

If not I think we maybe need to modify one of these examples to expose the issue. What change would we need to make? Is it a case of just turning one of the author names into a link?

**Comment by kata-lin** [comment]
Yes, in order to replicate it in the storybook the table will need to be updated so the cells for one of the columns contain a link each.
